### PR TITLE
Travel fund request for @maddhruv at Collab Summit Berlin

### PR DESCRIPTION
**Pull request by maddhruv**
Hey everyone,
This is a travel fund request for **Dhruv Jain (@maddhruv)** at **Collab Summit Berlin 2018**.
Event - Collaborator Summit Spring 2018
Location - Berlin, Germany
From - New Delhi, India
Date - May 31 - June 3
Amount - USD $1000
@nodejs/community-committee 

I am a member of @nodejs/website-redesign and @nodejs/i18n teams and am very excited to join the summit and share my thoughts with the Node people.
I expect a ton to learn from everyone there. I've never participated in any Node.js Foundation event.

I want to discuss new scopes for Website of the *foundation* and internationalization of *Node.js*.

The reimbursement stipend includes - $400 each for arriving and departing trips (economy class flight) and $200 for a hotel (dorm) and food in Berlin.

Wish to meet you soon :relaxed: :relaxed: :relaxed:

Regards

### CODE REVIEW

**Comment by maddhruv** [comment]
I have registered into it
And have also received mail from *The Linux foundation*

File: MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Ruben Bridgewater | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver
 Mihai Ene-Pietrosanu | Index 2018 | Collaborator & Learn | Portland, OR, US | 20Feb 2018 | US$ 283
 Joyee Cheung | Diagnostics WG Summit | Attendance | Ottawa, ON, CA | 12 Feb - 13 Feb 2018 | US$ 1650
 Benjamin Gruenbaum | Collab Summit Spring 2018 | Attendance & Code & Learn Mentor | Berlin, Germany | May 31 - Jun 3 2018 | $400
+Dhruv Jain | Collab Summit Berlin 2018 | Attendance & Code & Lean Mentor | Berlin, Germany | May 31 - June 3 2018 | $1000
```
[the remark this answers is not in the document]
Comment (maddhruv) [reply MEMBER_TRAVEL_FUND.md]: Fixed

**Comment by hackygolucky** [comment]
@maddhruv can you resolve the conflicts on this branch? After you do, ping me and we'll approve to merge!

**Comment by maddhruv** [comment]
@hackygolucky done

**Comment by maddhruv** [comment]
Thanks @hackygolucky for merging

**Comment by maddhruv** [comment]
> Once the request has been approved, provide receipts as attachments in an email stating your name, the participation covered, and the total approved for reimbursement

Do I understand this PR merge as *my request has been accepted for funds* and I can book the tickets?
Or there are some more stairs to climb!
Thanks

**Pull request merged**